pkg/http/server.go

```diff
@@ -104,6 +125,14 @@ func (s *server) setupTLSListener(opts Opts) error {
 	}
 	config.Certificates = []tls.Certificate{cert}
 
+	if len(opts.TLS.CipherSuites) > 0 {
+		cipherSuites, err := parseCipherSuites(opts.TLS.CipherSuites)
+		if err != nil {
+			return err
```

Comment:
Please always wrap errors with additional context, it helps debugging when there's an issue
```suggestion
			return fmt.Errorf("could not parse cipher suites: %v", err)
```